services/app-api/handlers/reports/release.test.ts

```diff
@@ -36,13 +37,22 @@ const releaseEvent: APIGatewayProxyEvent = {
 };
 
 describe("Test releaseReport method", () => {
-  afterEach(() => {
+  beforeEach(() => {
     jest.clearAllMocks();
+    dynamoClientMock.reset();
   });
 
   test("Test release report passes with valid data", async () => {
     mockAuthUtil.hasPermissions.mockReturnValueOnce(true);
-    mockDocumentClient.get.promise.mockReturnValueOnce({
+    // s3 mocks
+    const s3GetSpy = jest.spyOn(s3Lib, "get");
+    s3GetSpy
+      .mockResolvedValueOnce(mockReportJson)
+      .mockResolvedValueOnce(mockReportFieldData);
```

Comment:
Thought: This is an instance of what I alluded to in an earlier comment, where I think it would be preferable to mock the different return values based on different parameters (or possibly, to move the actual queries elsewhere, to be mocked by name here).

But of course, that's much harder to do here, because jest doesn't (afaik) provide such an easy route to match mocks to parameters. So this is totally fine as-is for now.